DalamudPlugin/RaidsRewritten/Scripts/Conditions/Condition.cs

```diff
@@ -9,6 +9,8 @@ namespace RaidsRewritten.Scripts.Conditions;
 public class Condition(ILogger logger) : ISystem
 {
     public record struct Component(string Name, float TimeRemaining);
+    public record struct Hidden();
+    public record struct IgnoreOnDeath();
```

Comment:
These parameter-less structs can just be declared as
```suggestion
    public struct Hidden;
    public struct IgnoreOnDeath;
```

since `record` is just C# shorthand for a struct with a constructor to populate its field variables